### PR TITLE
Implement format_check_dependencies method in simple formatter

### PR DESCRIPTION
**Pull request by chipp**
#162

### CODE REVIEW

File: spec/xcpretty/term_spec.rb [hunk 1]
```diff
@@ -4,22 +4,22 @@ module XCPretty
   describe Term do
     it "marks unicode as unavailable" do
       Encoding.stub(:default_external).and_return(Encoding::ASCII)
-      Term.unicode?.should be_false
+      Term.unicode?.should be_falsey
```
Comment (supermarin) [inline spec/xcpretty/term_spec.rb]: Could you please use `be false` instead?

File: spec/xcpretty/term_spec.rb [hunk 2]
```diff
@@ -4,22 +4,22 @@ module XCPretty
...
     end
 
     it "marks unicode as available" do
       Encoding.stub(:default_external).and_return(Encoding::UTF_8)
-      Term.unicode?.should be_true
+      Term.unicode?.should be_truthy
```
Comment (supermarin) [inline spec/xcpretty/term_spec.rb]: Same comment, `be true`

**Comment by supermarin** [comment]
Thanks for your contribution @chipp!

:+1: after comments

File: spec/xcpretty/term_spec.rb (hunk 1 above)
Comment (chipp) [inline spec/xcpretty/term_spec.rb]: I just fixed rspec warnings. https://github.com/carrierwaveuploader/carrierwave-mongoid/pull/146

Comment (supermarin) [inline spec/xcpretty/term_spec.rb]: @chipp sure, they can be fixed by replacing `be_false` with either `be false` or `be_falsey`.

I already did it in `master` yesterday and merged your commit.
Thanks!

**Pull request merged**